Bookmark-Manager/src/main/java/com/bookmarkmanager/bookmark/BookmarkService.java

```diff
@@ -1,36 +1,32 @@
 package com.bookmarkmanager.bookmark;
 
 import com.bookmarkmanager.bookmarkfolder.FolderRepository;
+import com.bookmarkmanager.exception.BookmarkManagerException;
 import com.bookmarkmanager.exception.BookmarkNotFoundException;
 import com.bookmarkmanager.exception.ResourceNotFoundException;
 import com.bookmarkmanager.pojo.Bookmark;
 import com.bookmarkmanager.pojo.Folder;
 import lombok.RequiredArgsConstructor;
 
 import java.time.LocalDateTime;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class BookmarkService {
+  public static final String FIELD = "field";
+  public static final String MESSAGE = "message";
+  public static final String ERROR_CODE = "errorCode";
   private final BookmarkRepository bookmarkRepository;
   private final FolderRepository folderRepository;
 
 
   public Bookmark addBookmark(Bookmark bookmark) {
```

Comment:
```suggestion
  public Bookmark createBookmark(Bookmark bookmark) {
```
This operation should be transactional, so use @Transactional

Reply:
Added those Changes